.github/workflows/test-on-iotlab.yml

```diff
@@ -110,9 +110,13 @@ jobs:
             -l 1,site=${{ matrix.boards.iotlab.site }}+archi=${{ matrix.boards.iotlab.archi }})
           iotlab-experiment wait -i ${IOTLAB_EXP_ID}
           echo "::set-env name=IOTLAB_EXP_ID::${IOTLAB_EXP_ID}"
+      - name: Install Python dependencies
```

Comment:
There is already a step that installs the Python dependencies: https://github.com/RIOT-OS/RIOT/pull/14871/files#diff-489558d2c142b44b698f71446b44a34cR84. Just add `junit-xml` to the existing list.

Reply:
Oops. Will do!

Reply:
Fixed and started another test run at https://github.com/miri64/RIOT/actions/runs/228647748